server/app/services/openApi/v2/Parameter.java

```diff
@@ -0,0 +1,89 @@
+package services.openApi.v2;
+
+import com.google.auto.value.AutoValue;
+import java.util.Optional;
+
+/**
+ * Describes a single operation parameter.
+ *
+ * <p>https://swagger.io/specification/v2/#parameter-object
+ */
+@AutoValue
+public abstract class Parameter {
+  /**
+   * Required.
+   *
+   * <p>The name of the parameter. Parameter names are case sensitive.
+   *
+   * <p>If in is "path", the name field MUST correspond to the associated path segment from the path
```

Comment:
typo?

```suggestion
   * <p>If in itss "path", the name field MUST correspond to the associated path segment from the path
```

Reply:
No... poorly rendered from [source](https://swagger.io/specification/v2/#parameter-object) 
![image](https://github.com/civiform/civiform/assets/195162/9631e117-b8ce-4c96-8dd4-43d4875514bb)

Reply:
@avaleske Does it make more to nix the text comments I pulled in and just have a link to the spec at the class level?

Reply:
ahh. At this point in the review I didn't know that `in` and `path` were additional concepts, and I didn't go back over my comments before submitting.

I think the text comments are helpful and we should keep them. Would adding formatting help, or make it harder to search?

Reply:
@avaleske I can add some formatting, but if I add formatting then you won't be able to just select the text and search the spec and if it ever gets copied over again we'll lose changes if no one notices.

Reply:
Ah... tbh it hadn't occurred to me to do that in the first place, so up to you. Maybe linking to the heading in the spec page would be more clear? but also a fair amount of work.